ocs_ci/ocs/ocp.py

```diff
@@ -238,6 +238,7 @@ def exec_oc_cmd(
             config.switch_ctx(original_context)
         return out
 
+    @retry(CommandFailed, tries=30, delay=3, backoff=1)
```

Comment:
30 retries seems to be a lot to me no?  I prefer max 3 attempts with delay about 30 seconds maybe?

Reply:
done